File: app/src/main/java/com/ad4th/seoulandroid/api/user/Coin.java

package com.ad4th.seoulandroid.api.user;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.math.BigInteger;

public class Coin {

    @SerializedName("tokenName")
    @Expose
    private String tokenName;
    @SerializedName("tokenSymbol")
    @Expose
    private String tokenSymbol;
    @SerializedName("balance")
    @Expose
    private BigInteger balance;
    @SerializedName("contractAddress")
    @Expose
    private String contractAddress;

    public String getTokenName() {
        return tokenName;
    }

    public void setTokenName(String tokenName) {
        this.tokenName = tokenName;
    }

    public String getTokenSymbol() {
        return tokenSymbol;
    }

    public void setTokenSymbol(String tokenSymbol) {
        this.tokenSymbol = tokenSymbol;
    }

    public BigInteger getBalance() {
        return balance;
    }

    public void setBalance(BigInteger balance) {
        this.balance = balance;
    }

    public String getContractAddress() {
        return contractAddress;
    }

    public void setContractAddress(String contractAddress) {
        this.contractAddress = contractAddress;
    }

}

File: app/src/main/java/com/ad4th/seoulandroid/api/intro/Coin.kt
package com.ad4th.seoulandroid.api.intro

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName
import java.math.BigInteger

class Coin {
    @SerializedName("name")
    @Expose
    var name: String? = null
    @SerializedName("symbol")
    @Expose
    var symbol: String? = null
    @SerializedName("decimals")
    @Expose
    var decimals: BigInteger? = null
    @SerializedName("initialSupply")
    @Expose
    var initialSupply: BigInteger? = null
    @SerializedName("contractAddress")
    @Expose
    var contractAddress: String? = null
    @SerializedName("coinType")
    @Expose
    var coinType: String? = null
    @SerializedName("administrator")
    @Expose
    var administrator: String? = null
    @SerializedName("transferAmount")
    @Expose
    var transferAmount: BigInteger? = null
    @SerializedName("sequence")
    @Expose
    var sequence: Int? = null
    @SerializedName("id")
    @Expose
    var id: Int? = null
    @SerializedName("createdAt")
    @Expose
    var createdAt: String? = null
    @SerializedName("updatedAt")
    @Expose
    var updatedAt: String? = null

}